content/guides/output-management.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Output Management
+contributors:
+  - skipjack
+---
+
+Managing webpack's [output]() and including it in your HTML files may not seem tough at first, but once you start [using hashes in filenames]() and outputting [multiple bundles](), things can start to get a bit hairy. However, there's no need to fear as a few plugins exist that will make this process much easier to manage.
+
+First let's take a look at where you might stand without these plugins:
+
+__index.html__
+
+``` html
+<!doctype html>
+
+<html>
+  <head>
+    <title>Output Management</title>
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="stylesheet" href="/styles.min.css" />
+    <script src="/vendor.bundle.js"></script>
+  </head>
+  <body>
+    <script src="/app.bundle.js"></script>
+  </body>
+</html>
+```
+
+Here we've loaded a favicon, our stylesheet (extracted with the [extract-text-webpack-plugin](/plugins/extract-text-webpack-plugin)), any libraries (split into [a separate bundle](/guides/code-splitting-libraries)), and finally our main bundle (`app.bundle.js`). This is ok, but what happens if we change our entry point names? What if we decide to take advantage of [better caching](/guides/caching)?
```

Comment:
According to https://webpack.js.org/writers-guide/#typesetting, plugins are enclosed in backticks and camel-cased, such as `ExtractTextWebpackPlugin`.